src/main/java/core/basesyntax/HelloWorld.java

```diff
@@ -1,9 +1,55 @@
 package core.basesyntax;
 
-/**
- * Feel free to remove this class and create your own.
- */
+import core.basesyntax.dao.FileReader;
+import core.basesyntax.dao.FileWriter;
+import core.basesyntax.dao.impl.FileReaderImpl;
+import core.basesyntax.dao.impl.FileWriterImpl;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.DataConverter;
+import core.basesyntax.service.OperationHandler;
+import core.basesyntax.service.OperationStrategy;
+import core.basesyntax.service.ReportGenerator;
+import core.basesyntax.service.ShopService;
+import core.basesyntax.service.impl.BalanceOperation;
+import core.basesyntax.service.impl.DataConverterImpl;
+import core.basesyntax.service.impl.OperationStrategyImpl;
+import core.basesyntax.service.impl.PurchaseOperation;
+import core.basesyntax.service.impl.ReportGeneratorImpl;
+import core.basesyntax.service.impl.ReturnOperation;
+import core.basesyntax.service.impl.ShopServiceImpl;
+import core.basesyntax.service.impl.SupplyOperation;
+import core.basesyntax.storage.FruitStorage;
+import core.basesyntax.storage.Storage;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
 public class HelloWorld {
-    // HINT: In the `public static void main(String[] args)` it is better to create instances of your classes, 
-    // and call their methods, but do not write any business logic in the `main` method!
+
+    public static void main(String[] args) {
+        FruitStorage storage = new Storage();
+
+        FileReader fileReader = new FileReaderImpl();
+        List<String> inputReport = fileReader.read("reportToRead.csv");
```

Comment:
Ensure that the file path "reportToRead.csv" is correct and accessible. Consider using a configuration file or environment variable for file paths.